### PR TITLE
Add documentation for NPPM_GETDOCUMENTFROMBUFFER.

### PR DESCRIPTION
**Pull request by Kered13**
This is documentation for PR #9085 in the notepad-plus-plus repo.

### CODE REVIEW

File: content/docs/plugin-communication.md [hunk 1]
```diff
@@ -630,6 +630,21 @@ User is responsible to allocate a buffer which is large enough.*
...
+
+*wParam [in]*
+: int bufferID
+
+*lParam [in]*
+: int, must be zero
+
+**Return value**:
+: Returns NULL if bufferID does not exist, otherwise a document pointer that can be used with SCI_SETDOCPOINTER.
+
```
Comment (pryrt) [inline content/docs/plugin-communication.md:647]: Please add a the `---` between your section and the next section to make it consistent with the rest of the document.

**Review by pryrt — CHANGES_REQUESTED** [review]
please edit

**Comment by donho** [comment]
I will merge it (or reject it).

**Comment by donho** [comment]
For the sake of security and stability, `Document` object should keep internal usage only.
This PR is rejected because #9085 is rejected.

**Pull request closed without merging**